returnn/config.py

```diff
@@ -128,7 +131,7 @@ def serialize(self):
         config_lines = []
         unreadable_data = {}
 
-        pp = pprint.PrettyPrinter(indent=2, width=150)
+        pp = pprint.PrettyPrinter(indent=2, width=150, sort_dicts=self.sort_dicts)
```

Comment:
This breaks for pythons < 3.8. how about passing arbitrary arguments to PrettyPrinter, i.e. change `sort_dicts` to `pprint_kwargs`. Then you can set it in your setup, but it will still work with older pythons for setups that don't use this option.

Reply:
Yes, I think that's a good idea. I'll update the PR accordingly.